cli/client/cluster.go

```diff
@@ -9,12 +9,13 @@ import (
 	"github.com/coreos/etcd/clientv3"
 )
 
-const clusterMembersBasePath = "/cluster/members"
+var clusterMembersPath = createBasePath(coreAPIGroup, coreAPIVersion, "cluster/members")
```

Comment:
You could also say `createBasePath(coreAPIGroup, coreAPIVersion, "cluster", "members")`, which I think is more in line with the intent of `createBasePath()`?